core/src/main/java/me/aurium/beetle/api/task/sync/coordination/SyncConstants.java

```diff
@@ -0,0 +1,50 @@
+package me.aurium.beetle.api.task.sync.coordination;
+
+import java.util.concurrent.CompletableFuture;
+
+public class SyncConstants {
+
+    private static final Object EMPTY = new Object();
+
+    public static <T> T CONSTANT() {
+        return (T) EMPTY;
+    }
+
+    public void cringe(CompletableFuture<Integer> future) {
```

Comment:
What does this do?

Reply:
i don't think it does anything and i'm pretty sure i was testing things